### PR TITLE
formatted level.js ref #208

### PR DESCRIPTION
**Pull request by YashKumarVerma**
formatted file
ref #208

### CODE REVIEW

**Comment by YashKumarVerma** [comment]
@niccokunzmann

File: js/level.js [hunk 1]
```diff
@@ -96,29 +36,102 @@ function Level(specification) {
...
         });
     };
     var me = this;
     this.startTile = null;
-    this.tiles = specification.map(function (row, y) {
-        return row.map(function (tileSpecification, x) {
+    this.tiles = specification.map(function(row, y) { // defined in /js/tiles.js
+        /* global NullTile */
+
+        const tileWidth = 429.544; // px, real width
```
Comment (niccokunzmann) [inline js/level.js]: Is this a code duplication? This is not just a formatting.

Comment (YashKumarVerma) [reply js/level.js]: couldn't get you ?

Comment (niccokunzmann) [reply js/level.js]: This part is a copy of the start. I will have a look at the file now.

Comment (niccokunzmann) [reply js/level.js]: Yep: https://github.com/YashKumarVerma/labyrinth/blob/5decafb2ccc6880d54869f5147d693f085f5d2bf/js/level.js#L63

is the same as 
https://github.com/YashKumarVerma/labyrinth/blob/5decafb2ccc6880d54869f5147d693f085f5d2bf/js/level.js#L3

You copied parts over to other locations.

Comment (niccokunzmann) [reply js/level.js]: This is not a formatting in my eyes.

Comment (YashKumarVerma) [reply js/level.js]: whops !

Comment (niccokunzmann) [reply js/level.js]: What do you think?

Comment (YashKumarVerma) [reply js/level.js]: https://github.com/fossasia/labyrinth/pull/216

Comment (YashKumarVerma) [reply js/level.js]: i'm using github to do the changes, so sorry for opening another pr.

**Comment by niccokunzmann** [comment]
If this one is a duplicate now, please close it.

**Pull request closed without merging**